### PR TITLE
[test] Place test constants in travis.xml.dist and travis.xml.travis

### PR DESCRIPTION
**Pull request by Maks3w**
Close #7383

### CODE REVIEW

File: tests/phpunit.xml.dist [hunk 1]
```diff
@@ -32,6 +32,451 @@
         <ini name="date.timezone" value="UTC"/>
 
         <!-- Integration Test Variables -->
+        <!--
+          Zend\Auth\Adapter\DbTable tests
+        -->
+        <cons name="TESTS_ZEND_AUTH_ADAPTER_DBTABLE_PDO_SQLITE_ENABLED" value="false"/>
```
Comment (weierophinney) [inline tests/phpunit.xml.dist]: These should be "const", not "cons".

File: tests/phpunit.xml.travis [hunk 2]
```diff
@@ -0,0 +1,510 @@
...
+    </filter>
+
+    <php>
+        <ini name="date.timezone" value="UTC"/>
+
+        <!-- Integration Test Variables -->
+        <!--
+          Zend\Auth\Adapter\DbTable tests
+        -->
+        <cons name="TESTS_ZEND_AUTH_ADAPTER_DBTABLE_PDO_SQLITE_ENABLED" value="true"/>
```
Comment (weierophinney) [inline tests/phpunit.xml.travis]: Same comment: `s/cons/const/` in all of these declarations.

**Comment by Maks3w** [comment]
@weierophinney Ready for merge

**Comment by Maks3w** [comment]
@weierophinney I've removed `TEST_ZEND_DB` constants.

**Pull request closed without merging**